lib/oneview-sdk/image-streamer/resource/api300/artifact_bundle.rb

```diff
@@ -0,0 +1,175 @@
+        # @param [OneviewSDK::ImageStreamer::Client] client The client object for the Image Streamer appliance
+        # @param [DeploymentGroup] deployment_group The DeploymentGroup with name or uri
+        # @return [Hash] The result hash with DeploymentGroup data
+        def self.create_backup(client, deployment_group)
+          ensure_resource!(deployment_group)
+          response = client.rest_post(BACKUPS_URI, 'body' => { 'deploymentGroupURI' => deployment_group['uri'] })
+          client.response_handler(response)
+        end
+
+        # Creates a backup bundle from the zip file
+        #   If there are any artifacts existing, they will be removed before the extract operation.
+        # @param [OneviewSDK::ImageStreamer::Client] client The client object for the Image Streamer appliance
+        # @param [String] file_path
+        # @param [String] artifact_name The name for the artifact that will be created
+        # @param [Integer] timeout The number of seconds to wait for completing the request. Default is 300.
+        # @return [Hash] The result hash with DeploymentGroup data
+        def self.create_backup_from_file!(client, deployment_group, file_path, artifact_name, timeout = OneviewSDK::Rest::READ_TIMEOUT)
+          ensure_resource!(deployment_group)
+          params = { 'name' => artifact_name, 'deploymentGrpUri' => deployment_group['uri'] }
+          client.upload_file(file_path, BACKUPS_ARCHIVE_URI, params, timeout)
+        end
+
+        # Download the backup bundle
+        # @param [OneviewSDK::ImageStreamer::Client] client The client object for the Image Streamer appliance
+        # @param [String] local_drive_path The path where file will be saved
+        # @param [String] artifact_bundle_backup The backup ArtifactBundle with 'downloadURI'
+        # @return [Boolean] true if backup was downloaded
+        def self.download_backup(client, local_drive_path, artifact_bundle_backup)
+          raise IncompleteResource, "Missing required attribute 'downloadURI'" unless artifact_bundle_backup['downloadURI']
+          client.download_file(artifact_bundle_backup['downloadURI'], local_drive_path)
+        end
+
+        # Method is not available
+        # @raise [OneviewSDK::MethodUnavailable] method is not available
+        def update(*)
+          unavailable_method
+        end
+
+        # Update the name of Artifact Bundle
+        # @param [String] new_name Name to update the Artifact Bundle
+        # @return [Boolean] if name was updated
+        # @raise [OneviewSDK::IncompleteResource] if the client or uri is not set
+        def update_name(new_name)
+          ensure_uri
+          response = @client.rest_put(@data['uri'], 'body' => { 'name' => new_name, 'type' => 'ArtifactsBundle' })
+          @client.response_handler(response)
+          @data['name'] = new_name
+          true
+        end
+
+        # Extracts the exisiting backup bundle on the appliance and creates all the artifacts.
```

Comment:
`# Extracts the exisiting` replace by `# Extracts the existing`. Fix this please.